guides/common/modules/con_overview-of-the-project-api.adoc

```diff
@@ -3,6 +3,9 @@
 [id="overview-of-the-{project-context}-api_{context}"]
 = Overview of the {Project} API
 
+[role="_abstract"]
+The {Project} REST API offers several benefits and supports various use cases for integrating the API into your workflows.
```

Comment:
```suggestion
The {Project} API offers several benefits and supports various use cases for integrating the API into your workflows.
```

To align with title/file name.

Maybe also reword line 9 from "the REST API" to "the {Project} API".

AFAIK there's also GraphQL but there are talks about its demise in the Foreman Community Forum.